express-get-json/server.js

```diff
@@ -0,0 +1,36 @@
+import express from 'express';
+
+const app = express();
+
+app.get('/api/grades', (req, res) => {
+  const array = [];
+  for (const properties in grades) {
+    array.push(grades[properties]);
+  }
+  res.json(array);
+});
+
+app.listen(8080, () => {
+  console.log('Listening for port 8080.');
+});
+
+const grades = {
```

Comment:
Usually you put constants at the top of the file. I guess that's just convention, though, that comes from the days when you couldn't reference constants before they were defined.